src/qt/utilitydialog.cpp

```diff
@@ -33,29 +33,46 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
     ui(new Ui::HelpMessageDialog)
 {
     ui->setupUi(this);
-
+    this->resize(500,0);//force view height to stretch
+    ui->aboutMessage->resize(0,0);//force label to stretch
     QString version = QString{PACKAGE_NAME} + " " + tr("version") + " " + QString::fromStdString(FormatFullVersion());
 
-    if (about)
-    {
-        setWindowTitle(tr("About %1").arg(PACKAGE_NAME));
-
         std::string licenseInfo = LicenseInfo();
-        /// HTML-format the license message from the core
+        std::string copyrightInfo = CopyrightInfo();
+        // HTML-format aboutMessage elements
+        QString versionInfoHTML = QString::fromStdString(FormatVersion());
+        QString copyrightInfoHTML = QString::fromStdString(CopyrightInfo());
+        QString aboutMessageInfoHTML = QString::fromStdString(AboutMessageInfo());
         QString licenseInfoHTML = QString::fromStdString(LicenseInfo());
         // Make URLs clickable
         QRegExp uri("<(.*)>", Qt::CaseSensitive, QRegExp::RegExp2);
         uri.setMinimal(true); // use non-greedy matching
+        versionInfoHTML.replace(uri, "<a href=\"\\1\">\\1</a>");
```

Comment:
why does it create a link here?

Reply:
You are correct! This string no longer needs this transform.
Awesome!